helm/cluster-shared/templates/_coredns.tpl

```diff
@@ -256,15 +256,22 @@ data:
               Runs at cluster creation to label and annotate default, kubeadm installed CoreDNS
               resources so they can be managed by Helm and replaced with our managed app.
         spec:
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 1000
+            fsGroup: 1000
           activeDeadlineSeconds: 1800 # 30 minutes
           restartPolicy: Never
           serviceAccountName: coredns-adopter
           tolerations:
           - operator: Exists
-          hostNetwork: true # No need to wait for CNI to be ready
```

Comment:
I guess we can turn it back on and create an exception for it

Reply:
do you know how we can test the changes made here?

Reply:
Hey @AverageMarcus, can you help me test this?

Reply:
Based on @T-Kukawka comment [here](https://github.com/giantswarm/roadmap/issues/3741#issuecomment-2449469327) it might not be worth the effort. What do you think @T-Kukawka?